vendor/vendor.json

```diff
@@ -59,6 +59,12 @@
 			"revisionTime": "2016-09-30T00:14:02Z"
 		},
 		{
+			"checksumSHA1": "DYv6Q1+VfnUVxMwvk5IshAClLvw=",
```

Comment:
Yes, I was wondering about that as well, but put it down to my own lack of experience with govendor. That would also explain why the version bundled was over 2 years old, since that's about how long ago logrus gained Solaris support.